Sources/TuistCache/ContentHashing/TargetContentHasher.swift

```diff
@@ -85,7 +85,6 @@ public final class TargetContentHasher: TargetContentHashing {
         let coreDataModelHash = try coreDataModelsContentHasher.hash(coreDataModels: graphTarget.target.coreDataModels)
         let targetActionsHash = try targetActionsContentHasher.hash(targetActions: graphTarget.target.actions)
         let dependenciesHash = try dependenciesContentHasher.hash(graphTarget: graphTarget, hashedTargets: &hashedTargets)
-        let environmentHash = try contentHasher.hash(graphTarget.target.environment)
         var stringsToHash = [
```

Comment:
is this intended? To maintain the previous behaviour we'd need to continue hashing the environment values :/

Reply:
Let's make EnvironmentVariable conform to hashable so it is synthesized by the compiler and we do not need the map here :)

Reply:
> Let's make EnvironmentVariable conform to hashable so it is synthesized by the compiler and we do not need the map here :)

Oh yeah, I missed that one.

Reply:
Hey @adellibovi,
I don't think I understand this. So, I got the `graphTarget.target.environmentVariables.hashValue` by conforming the `EnvironmentVariable` to the `Hashable` protocol. Where should I put it?

Reply:
Oops, you are right @setoelkahfi, that is not needed, got confused by Swift hash vs our Content hash.
Though it seems to me we would need to hash all three properties instead of just name, as changing one of those should invalidate the cache, @kwridan can you confirm?

Reply:
@adellibovi What I can think of right now is to convert the environment variable array into a single string and then hash it with the content hasher.